editor/src/messages/tool/common_functionality/shape_editor.rs

```diff
@@ -201,6 +246,106 @@ impl ShapeState {
 		Some(())
 	}
 
+	/// Reinsert a control point at the same point to make its handles recalculate their positions based on a new constraint.
+	pub fn blink_manipulator_group(&self, group_data: &ManipulatorGroupData, responses: &mut VecDeque<Message>, document: &Document) -> Option<()> {
```

Comment:
"blink" is sort of a confusing name, I still don't really have an understanding of why that word was chosen because it doesn't convey the meaning I'd expect in this context (it doesn't happen across multiple frames so there's no "blinking" in any sense of the word I'd intuitively understand it).

Reply:
renamed and retooled this function to `smooth_selected_groups`.

Reply:
That sounds like a much more sensible name, thank you.